src/transformers/models/grounding_dino/modeling_grounding_dino.py

```diff
@@ -1425,12 +1427,20 @@ def forward(
 
         # Cross-Attention Text
         queries = self.with_pos_embed(hidden_states, position_embeddings)
+        text_encoder_attention_mask = text_encoder_attention_mask[:, None, None, :]
```

Comment:
This assumes it's not None

Reply:
https://github.com/huggingface/transformers/pull/30364/commits/ca3933d97808ebff556309486b0514d8d9db3d30